modules/README.md

```diff
@@ -259,6 +259,14 @@ Toolkit vpc module, use:
 [tfsubdir]: https://www.terraform.io/language/modules/sources#modules-in-package-sub-directories
 [daos-cluster.yaml]: ../community/examples/intel/daos-cluster.yaml
 
+#### Generic Git Modules
```

Comment:
nit: Does this also support GitHub? Maybe we should only document this method. We can still leave the other functionality intact but it would give us less surface area to maintain.

Reply:
It covers both, but I kept them documented separately since the simplified option is available with github (doesn't require the git:: prefix). Terraform and go-getter (the package we and terraform use) both also document them separately:
* https://www.terraform.io/language/modules/sources#generic-git-repository
* https://www.terraform.io/language/modules/sources#github

I could see making github a subsection of git potentially, but I personally prefer this layout for consistency.

Reply:
Ack.